allennlp/data/dataset_readers/seq2seq.py

```diff
@@ -56,6 +56,7 @@ def __init__(self,
                  target_token_indexers: Dict[str, TokenIndexer] = None,
                  source_add_start_token: bool = True,
                  delimiter: str = "\t",
+                 max_tokens: Tuple[int, int] = (1000, 1000),
```

Comment:
Should there be a flag to choose whether to truncate or skip long sentences.

Reply:
Having such a flag sounds reasonable. I'd suggest keeping a counter of how often this occurs (either truncating or skipping) and logging it at the end of `_read` so that it doesn't happen silently. (Also, I'd suggest resetting these at the top of `_read` otherwise the logging is confusing when a dataset reader is used multiple times, e.g. on both training and validation sets.) And regardless, there should be some way to disable the truncating or skipping entirely as I mentioned above.